atlasdb-config/src/main/java/com/palantir/atlasdb/http/NotCurrentLeaderExceptionMapper.java

```diff
@@ -29,6 +28,6 @@
 public class NotCurrentLeaderExceptionMapper implements ExceptionMapper<NotCurrentLeaderException> {
     @Override
     public Response toResponse(NotCurrentLeaderException exception) {
-        return Response.noContent().status(503).header(HttpHeaders.RETRY_AFTER, "0").build();
+        return ExceptionMappers.encode503ResponseWithRetryAfter(exception);
```

Comment:
Yep, the entity becomes the content of a message. This is added in `encode503ResponseInternal` (`ExceptionMappers:39`). I'll add some docs to clarify this